yarn.lock

```diff
@@ -10414,6 +10551,18 @@ glob@7.1.3:
     once "^1.3.0"
     path-is-absolute "^1.0.0"
 
+glob@7.1.6, glob@^7.0.3, glob@^7.1.1, glob@^7.1.2, glob@^7.1.3, glob@^7.1.4, glob@^7.1.6:
```

Comment:
combine with the one below?

Reply:
never mind, we need at least 2 versions of glob because we have `glob@7.1.6` and `glob@7.2.0` which do not accept ranges